1908-01-20.xml

```diff
@@ -0,0 +1,5143 @@
+                        ne croire à sucun relèvement appréciable tant que la situation n'est pas changée, la plupart abondent dans le sens d'une reprise durable de non plus seulement à la pure speculation mais basée sur des principes solides, tels que la baisse du taux d'escompte et l'amélioration géadrale qui en est la conséquence. Quoi qu'il en soit, pour le moment la tendance wat baussière et si les transactions demeurent en somme restreintes, on peut dire que c'est la seule raison pour laquelle on ait lieu de se montrer réservé.
+                        A remarquer, faisant contraste avec l'activité de ce jours derniers, on délaissement de l'A gricole et de la National Bank, qui cloturent sensiblement au même prix. Toutefois l'Agricole perd 1/89 à 7 23/32
+                        Les Jouissanoes Eaux du Caire perdent aussi 4 frocs i 174 et les Nongovich Hotels 1/2 livre à 13.
+                        En revanche, l'Obligation Crédit Foncier avaboe rapidement de 280 à 987 l'ancienne et de 253 256 la nouvelle, ainsi que les Lots Turcs de 166 a 170. LA Deferred Delta pro gresse également de 5 à 51/8, la Land Bank de 7 3/8 à 7 1/2, et la Banque d'Orient de 114 1/2 à 118.
+                        Le Crédit Franco Bgyptien, plus ou moins délaissé depuis des mois, a passé pour ainsi dire au premier rang des titres favoris. Ce sont, parait-il, les arbitragistes avec Paris qui achè tent ici pour revendre sur cette place ou le prix est sensiblement supérieur, 107 francs, contro 104 ici. On a cloturé ce matin à 4 3/16 ache teurs contre 4 1/82 Vendredi passé.
+                        Il y a eu un peu de demande en Investment qui gagne 1/89 3/8, en Hooker eten NewEgyptian; cette dernière se relève légèrement.
+                        A signaler encore une avance de 1/16 en Dairs I 13 7/16, de 1 franc en Cassa di Boonto a 52, de 1 16 en Anglo American Nilek 314 de 1/89 en Estates 11/39 ot on Building Lande
+                        1 1/4, et de penny et demi - Port Said Salt á 10,9; seule la Salt en Boda recale de 14,6 a 14/3 et l'Urbaine de 3 1/4 33/16.
+                        La légère reprise constatée ces jours derniers en United provenait de l'arrangement inter veva entre cette société et le groupe de créanciers, arrangement aux termes daquel ceux-ci acceptent en paiement des obligations 15% Cette transaction, en permettant à l'United d'aller de l'avant, naturellement occasionné une détente et les actionnaire ne dtoespèrent plas de voir le titre revenir sinon
+                        son ancienne cotation, du moins un niveau plus avantageux que l'actuel.
+                        
+                    </p>
+                    
+                </div>
+                <cb n="2"/>
+                <div type="item">
+                    <head>SHIPPING MOVEMENTS.
+                    </head>
+                    <p>ALEXANDRIA HARBOUR
```

Comment:
here and elsewhere, there are some problems with missing `<p>`. I suspect that you pasted the OCR into the text view rather than the author view? If you paste into author view, the paragraphs are created automatically.